packages/db/prisma/schema.prisma

```diff
@@ -3,7 +3,7 @@
 
 generator client {
   provider        = "prisma-client-js"
-  previewFeatures = ["tracing", "metrics"]
+  previewFeatures = ["relationJoins", "tracing", "metrics"]
```

Comment:
Are we using `relationJoins` in the end?

Reply:
No, I accidentally left it in while testing. Good catch.

It looks is less performant compared to the current Prisma relation strategy when you try to pull elements from the db with a large offset.